wgpu-core/src/command/transfer.rs

```diff
@@ -48,7 +48,7 @@ pub enum TransferError {
     #[error(transparent)]
     MissingTextureUsage(#[from] MissingTextureUsageError),
     #[error("Destination texture is missing the `RENDER_ATTACHMENT` usage flag")]
-    MissingRenderAttachmentUsageFlag(TextureId),
+    MissingRenderAttachmentUsageFlag,
```

Comment:
any way to preserve the extra error information of which texture had the issue? 🤔

Reply:
I fixed this one since it was easy but not `InvalidDimensionExternal` since lots of other variants of `TransferError` don't include the texture label. We should find a better mechanism to include the label for all remaining variants.